admin_manual/groupware/circles/app_overview.rst

```diff
@@ -0,0 +1,152 @@
+========
+Overview
+========
+
+Circles is a Nextcloud App that allow Nextcloud users to generate their own grouping of users and contacts.
+
+
+
+Types of Circles
+^^^^^^^^^^^^^^^^
+
+.. note::
+ Once the Circles app is enabled, your users can creates 2 types of Circles: `Circle` and `Personal Circle`.
+ There is no way to switch from `Circle` to `Personal Circle` and vice versa.
+ `Personal Circle` are really specific and will be treated separately.
+
+
+When creating a Circle, its default configuration makes it 'Closed' and `Hidden`, meaning than no one
+can see it or join it but the Circle can already be used as an entity to share files and content.
+The user that created the Circle is the first and only Member and will have Owner rights that allows him/her to:
+
+- add/remove Members,
+- edit their Level,
+- change the settings of the Circle,
+- change its configuration
```

Comment:
configuration of what?

Reply:
configuration of the Circle itself. 
- Settings is about name, description, limit to the number of members
- Configuration is about how if the Circle is visible, open, federated, ...

Reply:
> Settings is about name, description, limit to the number of members
> Configuration is about how if the Circle is visible, open, federated, ...

Alright. Then let's make that more clear to the reader.